third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.cpp

```diff
@@ -468,4 +468,20 @@ void TargetInfo::storeOpAnnotation(triton::gpu::LocalStoreOp op,
   storeOpSchedAnnotations(op, localStoreOpCount, type);
 }
 
+SmallVector<unsigned> TargetInfo::supportedDirectToLdsWidths() const {
```

Comment:
I think `bool supportsDirectTodsLoadBitwidth(int width)` is better--we don't need to always return a temporary `SmallVector` for it.